pages/partials/questionNavSideButtonGroup.ejs

```diff
@@ -0,0 +1,18 @@
+<div class="text-center">
+  <%- include("../partials/questionNavSideButton", {
+    button: {
+      "id": "question-nav-prev",
+      "label": "Previous question",
+      "sequenceLockedPopoverContent": "Earlier problems must be completed to go back.",
+      "question": instance_question_info.prev
+    }
+  }); %>
+  <%- include("../partials/questionNavSideButton", {
+    button: {
+      "id": "question-nav-next",
+      "label": "Next question",
+      "sequenceLockedPopoverContent": `You must score at least ${instance_question_info.min_advance_perc}% on this question to unlock the next.`,
```

Comment:
I think we need to be more precise with our language. There are two different concepts of "percentage score" that are easy to confuse:

1. The percentage score of a _submission_. Every submission gets a score from 0-100. This is the score we are using when determining whether we can advance to the next question, I believe?
2. The percentage score awarded for a _question_. That is, `points / max_points`. This is often called the "question percentage score" in PL, and this is not what we are using.

For purposes of question advancement, I think we should say "You must score at least ${instance_question_info.min_advance_perc}% on a submission to this question to unlock the next question."

Can we make similar changes everywhere else in the UI, and also in the docs?